includes/class-sensei-usage-tracking.php

```diff
@@ -261,6 +261,20 @@ public function send_usage_data() {
 	// Private functions
 	//
 
+	/**
+	 * Add two week schedule to use for cron job.
+	 *
+	 * @access private
```

Comment:
My intention here was to indicate that this function should not be called from external code, i.e. developers should consider it private, even though it is not.

It cannot actually be private, because it is being used by a filter, and filters can only call public methods. But logically, this method should be private.

As inspiration, this is what is used by WP Core for functions such as [`_get_cron_array`](https://core.trac.wordpress.org/browser/tags/4.9/src/wp-includes/cron.php#L477). The function itself cannot be private, but the documentation clearly marks it as private, so developers should not call it.

This is also why I prepended the underscore (as you called out [here](https://github.com/Automattic/sensei/pull/2015#discussion_r167246585)). The convention makes it clear that this function should be considered private, even though it is not.

I'm open to suggestions on other ways to solve this! The problem is that I want the methods to be private, but they have to be public.

Reply:
You'll notice that the functions in that WordPress file are written using procedural programming and not object-oriented programming. Since you don't have any access modifiers for procedural code, they're using the `_` thing to try to show that the function is meant to be private. But it's a redundant practice for OOP.

Regarding the modifier, I'd probably just do what we do elsewhere in Sensei.